tests/suites/test_suite_hmac_drbg.function

```diff
@@ -57,15 +58,18 @@ void hmac_drbg_entropy_usage( int md_alg )
     else
         default_entropy_len = 32;
 
+    /* Set reseed interval before seed */
+    mbedtls_hmac_drbg_set_reseed_interval( &ctx, (2 * reps) );
```

Comment:
`set_reseed_interval` is now tested only before `seed`. Please expand the function to test `set_reseed_interval` both before and after `seed`.

Goes for both DRBG.

Reply:
Removed my changes here as the original tests were sufficient.

Reply:
Sorry, no the original tests aren't sufficient. Please add test cases that fail with the old library code (non-regression tests).

Reply:
- Added test to check that DRBG reseeds when `reseed_counter` reaches `reseed_interval`, if `reseed_interval` set before `seed` and `reseed_interval` is less than `MBEDTLS_CTR_DRBG_RESEED_INTERVAL`.
- Modified function to test `set_reseed_interval` after `seed`.